docs/docs/configuration.md

```diff
@@ -572,6 +614,15 @@ Default: `dist`
 
 The directory to build your assets into when running `sku build` or `sku build-ssr`
 
+## testRunner
+
+Type: `string`
```

Comment:
Similar to the bundler one, I think this should specify that its a union

```suggestion
Type: `jest` | `vitest`
```

That would allow us to not need to specify the valid options below